rotkehlchen/chain/evm/decoding/types.py

```diff
@@ -2,6 +2,21 @@
 
 
 class CounterpartyDetails(NamedTuple):
+    """
+    details for a counterparty returned to the api consumer.
+    icon is used when the counterparty uses an icon from the icon package instead of
+    a preselected image.
```

Comment:
```suggestion
    Details for a counterparty returned to the api consumer.
    
    The icon is used when the counterparty uses an icon from the icon package instead of
    a preselected image.
```